graphql_api/types/commit/commit.graphql

```diff
@@ -1,8 +1,8 @@
 type Commit {
   state: String
   message: String
-  createdAt: DateTime
-  commitid: String
+  createdAt: DateTime!
```

Comment:
So you know how this field doesn't have an explicit resolver? That's because ariadne somehow ends up in your DB model and if it finds a 1:1 mapping with the resolver name and it's field, or through an alias like in this case, it figures it out. I'd like to try and get rid of every implicit resolver (not in this ticket of course) the same way we came to return type parity with this ticket. Could you make an issue that expresses that? 🙏

Reply:
Yeah this `createdAt` field maps to the `timestamp` column in `commits` table which is a not null field hence adding the `!` here. Will make a ticket for implicit resolvers.
https://github.com/codecov/engineering-team/issues/745